mobile/library/common/jni/jni_impl.cc

```diff
@@ -962,8 +962,8 @@ Java_io_envoyproxy_envoymobile_engine_JniLibrary_registerKeyValueStore(JNIEnv* e
   api->remove = jvm_kv_store_remove;
   api->context = retained_context;
 
-  envoy_status_t result = register_platform_api(env->GetStringUTFChars(name, nullptr), api);
-  return result;
+  Envoy::Api::External::registerApi(env->GetStringUTFChars(name, nullptr), api);
```

Comment:
I might have missed updating this code to use `JniHelper`. Can we use `JniHelper::getStringUtfChars`? https://github.com/envoyproxy/envoy/blob/226d37da07b1f015827b44f1dc093ec81d71b063/mobile/library/common/jni/jni_helper.cc#L65